src/views/HomeView.vue

```diff
@@ -297,15 +293,7 @@
           <p>{{ $t("partners.body") }}:</p>
           <br />
           <div class="container">
-            <carousel
-              :per-page-custom="[
-                [480, 2],
-                [768, 4],
-                [1024, 5],
-              ]"
-              :scroll-per-page="true"
-              pagination-active-color="#3273dc"
-            >
+            <carousel :breakpoints="carouselBreakpoints">
```

Comment:
When testing, I noticed that certain screen widths cause the elements in the carousel to have too much vertical space in between them.